docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2468,6 +2468,36 @@ definitions:
       authorization:
         $ref: '#/definitions/EventTypeAuthorization'
 
+      ordering_key_fields:
+        type: array
+        description: |
+          This is only an informational field. The events are delivered to consumers in the order they were published.
+          No reordering is done by Nakadi.
+
+          This field is useful in case the producer wants to communicate the complete order accross all the events
+          published to all partitions. This is the case when there is an incremental generator on the producer side,
+          for example.
+
+          It differs from `partition_key_fields` in the sense that it's not used for partitioning (known as sharding in
+          some systems). Also, the order of the events published in the same partition is not guaranteed to be
+          monotonically increased as with `partition_key_fields`. For `ordering_key_fields`, the consumer should handle
```

Comment:
> Also, the order of the events published in the same partition is not guaranteed to be monotonically increased as with `partition_key_fields`.

This sentence sounds as if the `partition_key_fields` indicates ordering (which it doesn't, as far as I know).
Maybe instead add this to the previous paragraph:

> The order indicated by the fields can differ from the order the events are in each partition, in case of out-of-order submission.

Another difference would be that this property applies to the whole event, not just the part inside of `data`, so it is possible to refer to metadata fields.

Reply:
@ePaul thanks, changed here 1804d4d

Regarding the possibility to access metadata fields, it's better described in the array item description.